src/script/components/MessagesList/Message/ContentMessage/asset/ImageAsset/ImageAsset.tsx

```diff
@@ -100,16 +100,15 @@ export const ImageAsset = ({
   });
 
   const imageContainerStyle: CSSObject = {
-    ...(!imageUrl?.url ? {maxWidth: '100%'} : {maxWidth: 'var(--conversation-message-asset-width)'}),
+    maxWidth: 'var(--conversation-message-asset-width)',
   };
 
   const imageAsset: CSSObject = {
     aspectRatio: isFileSharingReceivingEnabled ? `${asset.ratio}` : undefined,
     ...(!imageUrl?.url
       ? {
-          maxWidth: '100%',
-          width: asset.width,
-          height: asset.height,
+          maxWidth: asset.width,
+          maxHeight: asset.height,
```

Comment:
I think we can refactor to something like that 

```
{
  maxWidth: asset.width,
  maxHeigth: image.url ? '80vh' : asset.height
}
```

Reply:
done 3ed385cc7c257def9e75a8983d146d527daf614e